src/core/main.c

```diff
@@ -1253,37 +1253,39 @@ static void bump_file_max_and_nr_open(void) {
 #endif
 
 #if BUMP_PROC_SYS_FS_NR_OPEN
-        int v = INT_MAX;
+        unsigned int v = INT_MAX;
```

Comment:
We use `unsigned` rather than `unsigned int`.

Reply:
Deal. Addressed in 2c2a0bac86085278dd66918928f2143046c8b8b4 .